crowbar_framework/config/locales/crowbar/en.yml

```diff
@@ -442,6 +442,13 @@ en:
       powercycle: 'Power Cycle'
       poweroff: 'Power Off'
       identify: 'Identify'
+      blockui_reboot: 'Triggering reboot...'
+      blockui_shutdown: 'Triggering shutdown...'
+      blockui_poweron: 'Triggering poweron...'
+      blockui_powercycle: 'Triggering powercycle...'
+      blockui_poweroff: 'Triggering poweroff...'
+      blockui_identify: 'Triggering identify...'
```

Comment:
Can we have proper strings (and not states) in the strings? poweron -> "Power On", etc.